web-app/src/components/BookingCalendar/container.js

```diff
@@ -126,13 +127,19 @@ const BookingCalendarContainer = Wrapped =>
       }
     };
 
+    onNavigate = date => {
+      const newDate = new moment(date);
+      this.props.onNavigate(newDate);
```

Comment:
Naming is a bit weird here or I'm being silly. You have a function called onNavigate, inside you get a new formatted date and pass it to a new variable newDate. Then invoke the function onNavigate passing in the newDate variable? 

I'd be renaming either the prop or the function name personally.

Reply:
This actually isn't even needed at all after some changes I made later on. :)